dashboard/config/scripts/levels/AnjaliTest Sprite Lab Project.level

```diff
@@ -0,0 +1,88 @@
+    "auto_run_setup": "DRAW_LOOP",
+    "parent_level_id": 4578,
+    "show_type_hints": "true",
+    "include_shared_functions": "true",
+    "block_pool": "gamelab",
+    "block_pools": [
+      "GamelabJr",
+      "aalab"
```

Comment:
Intentional to also include the "aalab" library on your test level?

Reply:
Whoops, nope that was not intentional